lib/aruba/matchers/command.rb

```diff
@@ -0,0 +1,79 @@
+# @!method have_exit_status(status)
+#   This matchers checks if <command> has exit status <status>
+#
+#   @param [Integer] status
+#     The value of the exit status
+#
+#   @return [TrueClass, FalseClass] The result
+#
+#     false:
+#     * if command does not have exit status
+#     true:
+#     * if command has exit status
+#
+#   @example Use matcher
+#
+#     RSpec.describe do
+#       it { expect(last_command).to have_exit_status(0) }
+#     end
+RSpec::Matchers.define :have_exit_status do |expected|
+  match do |actual|
+    @old_actual = actual
+
+    @announcer ||= Aruba::Announcer.new(
```

Comment:
I tried to use `announcer` from `Aruba::Api`. This works fine in `RSpec`, but fails in `cucumber`. I'm not sure why, but I think it has something to how the binding is passed to the matcher. See rspec/rspec-expectations#803 for this. Now I use @announcer ... for this.